src/main/java/com/google/starfish/servlets/HandleNotesServlet.java

```diff
@@ -88,7 +158,6 @@ private boolean validateUser(HttpServletRequest req) {
     for (Cookie cookie : cookies) {
       if (COOKIE_NAME.equals(cookie.getName())) {
         sessionId = cookie.getValue();
-        break;
```

Comment:
Isn't this an addition from another PR review? There might be some changes that need to be merged.

Reply:
Ohh you are right, I think just adding the break back to this PR will prevent it from breaking anything on merge and keep what we want. Good catch!